tools/run_tests/python_utils/start_port_server.py

```diff
@@ -50,8 +51,7 @@ def start_port_server(port_server_port):
     print('detected port server running version %d' % version)
     running = True
   except Exception as e:
-    print('failed to detect port server: %s' % sys.exc_info()[0])
-    print(e.strerror)
+    print('failed to detect port server: %s' % e)
```

Comment:
[:-(](https://sadtrombone.com/?autoplay=true)

Might now be the time to address that? Okay to say no.